retrofit-adapters/rxjava/src/test/java/retrofit2/adapter/rxjava/ObservableTest.java

```diff
@@ -103,31 +97,32 @@
   @Test public void responseSuccess200() {
     server.enqueue(new MockResponse().setBody("Hi"));
 
-    BlockingObservable<Response<String>> o = service.response().toBlocking();
-    Response<String> response = o.first();
+    TestSubscriber<Response<String>> subscriber = new TestSubscriber<>();
+    service.response().subscribe(subscriber);
+    Response<String> response = subscriber.getOnNextEvents().get(0);
     assertThat(response.isSuccessful()).isTrue();
     assertThat(response.body()).isEqualTo("Hi");
+    subscriber.assertCompleted();
```

Comment:
I don't see why. `SafeSubscriber` dictates the `TestSubscriber` will only ever see a completed event or an error event, so the lack of a completed event will already fail the test.